packages/experimental/src/inbox-note/style.scss

```diff
@@ -174,22 +174,6 @@
 	}
 }
 
-.woocommerce-homepage-notes-wrapper {
```

Comment:
Removed css unrelated to the inbox note components and moved back to the inbox-panel/index.scss